src/astrobleme/gui/Main.java

```diff
@@ -41,8 +41,8 @@
  */
 public class Main extends Application {
 
-    private static final int WIDTH = 500;
-    private static final int HEIGHT = 500;
+    private static final int WIDTH = 200;
+    private static final int HEIGHT = 200;
```

Comment:
Why was this necessary? To make it fit on screen or for debugging?

It might also be best to have default values and allow the user to override it through environment variables. (Note that I _did_ write this code over 5 years ago...)

Reply:
Yes, small game panel helps debug by reducing time of making way for the snake to eat fruits.